docs/user/gettingStarted/GeneratingData.md

```diff
@@ -41,13 +41,13 @@ The generator can be used to generate data which intentionally violates the prof
 
 Using the `violate` command produces one file per rule violated along with a manifest that lists which rules are violated in each file.
```

Comment:
Perhaps we should mention again here that violation mode is an alpha feature

Reply:
I think this needs more discussion, and is out of scope of this PR. A ticket addressing this is currently high on the un-readied list: #1153